src/haddock/modules/refinement/flexref/__init__.py

```diff
@@ -57,6 +57,8 @@ def generate_flexref(
     output = f"{linesep}! Output structure{linesep}"
     output += (f"eval ($output_pdb_filename="
                f" \"{output_pdb_filename}\"){linesep}")
+    output += (f"eval ($count="
+               f" {identifier}){linesep}")
```

Comment:
Identifier here starts at 0

Reply:
Yes - this is still how models are numbered currently, starting at 0 instead of 1 (which might be more logical from a user perspective)